components/groups/windows/DisabledWindowItem.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <v-card
+    flat
+    tile
+  >
+    <v-window
+      v-model="onboarding"
+      show-arrows
+    >
+      <v-window-item
```

Comment:
What do you think about adding items not in loop and, for example, add 3 items with different text and make one of them disabled (so we can see that when we are using navigation arrows we'll skip disabled item)

Reply:
DONE